queries.sql

```diff
@@ -10,3 +10,45 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE NOT name = 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN  10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+SELECT * FROM animals;
+COMMIT;
```

Comment:
- [ ] Please 🙏🏽  add this line at the end of your query 👇🏽 

``SELECT species from animals;``

It is to verify that the change persists after COMMIT 👇🏽 

> Verify that changes persist after commit.

Reply:
Thank you for the feedback @franclobo
Will that replace line 17?